### DISCUSSION TITLE
Does it support validating arrays?

### DISCUSSION THREAD

**Discussion by dcordobag** [CacheControl/json-rules-engine#374] [General]
Hi there, I'm working with a JSON file and trying to validate the rows in my data, but don't find a way to handle with it, do you have some good example of how can i do it? like... if i have a table, i want to validate the field 1 in the second row, if the condition is met i want to retrieve the value of the field 2 (for example)

**Comment by chris-pardy** [comment]
@dcordobag This is likely not supported yet as there's no functionality for the rules engine to work on arbitrary length arrays. Are the number of fields / rows in your data known. If not then you'll need to move the iteration outside of the rules engine. Effectively running the rules engine for each row with the same rules.
